backend/routes/post.py

```
@@ -75,6 +79,59 @@ def get_post_additional_data(post: Post, db: Session):
+    post = query.first()
+    if not post:
+        raise HTTPException(status_code=404, detail=STATUS_404_ERROR)
+    return post
+
+
+def update_post_content(post: Post, content: Optional[str]):
+    if content is not None:
+        post.content = content
+
+
+def remove_old_file_if_exists(file_path: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `remove_old_file_if_exists`

Add return type annotation: `None`

(ANN201)